scripts/create-hidive-news.ts

```diff
@@ -43,6 +43,11 @@ let issueTemplateSchema = z.object({
     .optional()
     .transform((x) => x?.trim())
     .transform((x) => x === "" ? undefined : x),
+  date: z.string()
+    .nullable()
+    .optional()
+    .transform((x) => x?.trim())
+    .transform((x) => x === "" ? new Date().toISOString().split("T")[0] : x),
```

Comment:
I think there is a very subtle bug here. We only get a fallback data if the x is an empty string, but it could also be undefined. If someone did not provide a value in the form, I believe the output would be `undefined-{base}.md`.

```suggestion
    .transform((x) =>
      (x === undefined || x === "") ? new Date().toISOString().split("T")[0] : x
    ),
```